packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -24,6 +24,7 @@ export const AWS_CDK_METADATA = new Set([
   'eu-west-3',
   'eu-north-1',
   'eu-south-1',
+  'eu-south-2',
```

Comment:
```suggestion
  // 'eu-south-2',
```

We haven't added metadata here, i don't think